kable-core/src/appleMain/kotlin/logs/LogMessage.kt

```diff
@@ -19,21 +19,41 @@ internal fun LogMessage.detail(error: NSError?) {
     if (error != null) detail("error", error.toString())
 }
 
-internal fun LogMessage.detail(service: CBService) {
-    detail("service", service.UUID.UUIDString)
+internal fun LogMessage.detail(service: CBService? = null) {
+    detail("service", service?.UUID?.UUIDString ?: "Unknown UUID")
```

Comment:
Small nitpick: the other unknowns just say `Unknown`, but this one is `Unknown UUID` -- I don't have a preference for either one, but I'd prefer consistency.